web/template/index.html

```diff
@@ -23,21 +23,68 @@ <h2>Mainflux Deployment Summary</h2>
                 </ul>
             </div>
         </div>
-        <div class="container">
-            <div id="filter-container">
-                <form id="filter-form" onsubmit="applyFilter(event)">
-                    <label for="from-date">From:</label>
-                    <input type="date" id="from-date" name="from-date" value="{{.From}}">
-                    <label for="to-date">To:</label>
-                    <input type="date" id="to-date" name="to-date" value="{{.To}}">
-                    <input class="btn btn-primary" type="submit" value="Apply">
-                </form>
-            </div>
+         <!-- Modal -->
+        <div class="modal fade" id="applyFilters" tabindex="-1" aria-labelledby="applyFiltersLabel" aria-hidden="true">
+                    <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                        <h1 class="modal-title fs-5" id="applyFiltersLabel">Apply Filters</h1>
+                        </div>
+                        <div class="modal-body">
+	                        <div class="mb-3">
+                                <form id="filter-form" onsubmit="applyFilter(event)">
+                                    <div>
```

Comment:
```suggestion
```

Reply:
this has no styling attached, I don't see how it's used, remove it alongside, https://github.com/mainflux/callhome/pull/20#discussion_r1285491008